_data/collection-index.yml

```
@@ -358,3 +358,8 @@
   contact: https://github.com/GMkonan/feature/issues
   repository: https://github.com/GMkonan/feature
   ociReference: ghcr.io/gmkonan/feature
+- name: Dapr devcontainer features
+  maintainer: Dapr maintainers
+  contact: https://github.com/dapr/cli/issues
+  repository: https://github.com/dapr/cli
+  ociReference: ghcr.io/dapr/cli/dapr-cli
```

```suggestion
  ociReference: ghcr.io/dapr/cli
```

Hi 👋 

The `ociReference` here refers to the collection metadata artifact instead of the individual Features. Hence, the author needs to only add a single entry irrespective of the number of Features that might exist in their repository.

Can you help update this? thanks!

---

Updated - thanks @samruddhikhandale!